cmd/mesh/upgrade.go

```diff
@@ -239,11 +229,11 @@ func upgrade(rootArgs *rootArgs, args *upgradeArgs, l *Logger) (err error) {
 	return nil
 }
 
-// checkUpgradeValues checks the upgrade eligibility by comparing the current values with the target values
-func checkUpgradeValues(curValues, tarValues, ignoreValues string, l *Logger) {
-	diff := compare.YAMLCmpWithIgnore(curValues, tarValues, nil, ignoreValues)
+// checkUpgradeICPS checks the upgrade eligibility by comparing the current ICPS with the target ICPS
+func checkUpgradeICPS(curICPS, tarICPS, ignoreICPS string, l *Logger) {
+	diff := compare.YAMLCmpWithIgnore(curICPS, tarICPS, nil, ignoreICPS)
 	if diff == "" {
-		l.logAndPrintf("Upgrade check: Values unchanged. The target values are identical to the current values.\n")
+		l.logAndPrintf("Upgrade check: ICPS unchanged. The target values are identical to the current values.\n")
 	} else {
 		l.logAndPrintf("Upgrade check: Warning!!! The following values will be changed as part of upgrade. "+
 			"If you have not overridden these values, they will change in your cluster. Please double check they are correct:\n%s", diff)
```

Comment:
"If you have not overridden these values" doesn't seem to be applicable anymore because this is exactly what we're checking for.

Reply:
Fixed.